packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -3970,6 +3970,13 @@ export type ResourceFetchResult = {
     error?: Error;
 };
 
+// @beta
+export interface RichTextEditorOptions {
```

Comment:
Should `onPaste` be behind a CC flag `@conditional-compile-remove(rich-text-editor-image-upload) */`?

Reply:
`onPaste` is already under @conditional-compile-remove(rich-text-editor-image-upload) */
![image](https://github.com/Azure/communication-ui-library/assets/98852890/7ff63f80-16c3-4082-88b3-6d7d5fa7a5ed)

Reply:
Ah right, this is the MD file. No worries....